src/components/MostPopular/PopularItem/index.js

```diff
@@ -0,0 +1,19 @@
+import * as S from "./PopularItem.style"
+import Tag from "@/components/Tag"
+
+export default function PopularItem({ item }) {
+  return (
+    <S.Wrapper>
+      <Tag
+        text={item.category}
```

Comment:
You are passing the same value through 2 different props - `item.category` being passed to both `text` and `category` props
maybe you don't need to pass "text" you can just pass the tag's category and render the text using that.

Reply:
Smart boy! Prop `text` has been deprecated